docs/docs_source/hostpage.rst

```diff
@@ -90,6 +90,22 @@ sample host page:
     :emphasize-lines: 9, 11-29
 
 
+Working around browser iframe behavior
+--------------------------------------
+
+Some browsers exhibit strange behavior with iframes when using bookmarks or the forward/back browser buttons. In some
+cases, this will cause the Office Online iframe to be loaded twice in a single navigation. This in turn can cause
```

Comment:
Static frame use can also keep the frame from being recreated at all, causing the app to load with the previous session's state.